build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
let's call this `build-images.go` so it's clear what it is doing.  Want to add a make target too?